api/v1alpha1/helper.go

```diff
@@ -31,7 +31,7 @@ func GetServiceAccountName(etcdObjMeta metav1.ObjectMeta) string {
 
 // GetConfigMapName returns the name of the configmap for the Etcd.
 func GetConfigMapName(etcdObjMeta metav1.ObjectMeta) string {
-	return fmt.Sprintf("etcd-bootstrap-%s", string(etcdObjMeta.UID[:6]))
+	return fmt.Sprintf("%s-config-%s", etcdObjMeta.Name, (etcdObjMeta.UID[:6]))
```

Comment:
I think historically, the UID was used to identify if the configMap is in use by the running `Etcd`. Having UID will allow us to make cleaning operations of any orphan configMap's present in the cluster easily, as it's easy to find whether that configMap is currently being used or not. Although this isn't a strict requirement i.e we can still find without this but I feel having it would be better.

Yeah, I'm not sure why we haven't followed this UID suffix for other resources, but I'll just let them be for now.

> If you really want to keep the suffix then there is no need to enclose it in parenthesis.

Sure, somehow missed it.

Reply:
>the UID was used to identify if the configMap is in use by the running Etcd

I did not understand this reasoning. You can easily look at the owner reference, right? I would like that we have on convention and since we are changing the name, lets avoid suffixing a partial UID to it.

Reply:
> You can easily look at the owner reference, right?

yes yes, as I mentioned, `this isn't a strict requirement i.e we can still find without this but I feel having it would be better`, but yeah to maintain consistency, I can remove the suffix.